test/plugin/example/handler.go

```diff
@@ -5,14 +5,13 @@ import (
 	"errors"
 	"log"
 	"net"
-	"net/http"
 	"strings"
 	"time"
 
-	"golang.org/x/crypto/ssh/agent"
-
-	"github.com/conjurinc/secretless-broker/pkg/secretless/config"
 	plugin_v1 "github.com/conjurinc/secretless-broker/pkg/secretless/plugin/v1"
+	"github.com/conjurinc/secretless-broker/pkg/secretless/config"
+	"golang.org/x/crypto/ssh/agent"
+	"net/http"
```

Comment:
Builtin imports should be at the top. Third-party (non-conjurinc) import should be between built-ins and conjurinc imports.